docs/source/getting-started/installing.adoc

```diff
@@ -76,6 +76,11 @@ On the Windows operating system, due to issue link:https://github.com/minishift/
 You cannot run {project} from a network drive.
 ====
 
+[NOTE]
+====
+Minishift will use any `ssh.exe` found on the `PATH` in preference to internal ssh code. Ensure that any `ssh.exe` in the system `PATH` does not generate warning messages as this will cause installation problems such as issue  link:https://github.com/minishift/minishift/issues/1923[#1923].
+====
+
```

Comment:
@robin-owen  Can you please have a look on wording side?

Reply:
@karlnicholas we need to keep the information generic as it is applicable to all the supported operating systems. Here is my suggestion.

Minishift will use any available `ssh` binary found on the `PATH` in preference to the ssh implementation in the code. Ensure that the`ssh` binary in the system `PATH` does not generate warning messages when used by Minishift as this will cause installation problems such as issue  link:https://github.com/minishift/minishift/issues/1923[#1923].

Reply:
Ok, recommended changes have been made.